openapi.json

```diff
@@ -5831,11 +5621,5 @@
   "x-explorer-enabled": true,
   "x-proxy-enabled": true,
   "x-samples-enabled": false,
-  "x-samples-languages": [
-    "curl",
-    "node",
-    "ruby",
-    "javascript",
-    "python"
-  ]
-}
\ No newline at end of file
+  "x-samples-languages": ["curl", "node", "ruby", "javascript", "python"]
+}
```

Comment:
@amahadaya seems to be mostly formatting updates? Doesn't seem to have been in scope for this fix